[DOC] Note that Masker.inverse_transform only performs spatial unmasking in docstring (#3445)
nilearn/nilearn: doc/manipulating_images/masker_objects.rst

Inverse transform: unmasking data
---------------------------------

.. note::

  Inverse transform only performs spatial unmasking.

>   The data is only brought back into either a 3D or 4D represnetaion.

Reviewer: ```suggestion
  The data is only brought back into either a 3D or 4D representation.
```

>   The data is only brought back into either a 3D or 4D represenetation.

Reviewer 2: ```suggestion
  The data is only brought back into either a 3D or 4D representation,
  without inverting any signal processing performed by `transform`.
```

Inverse transform: unmasking data
---------------------------------

.. note::

  Inverse transform only performs spatial unmasking.
  The data is only brought back into either a 3D or 4D represenetation,

>   without inverting any signal processing performed by 'transform'.

Reviewer 3: ```suggestion
  without inverting any signal processing performed by ``transform``.
```
